draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -278,29 +278,100 @@ Signatureless certificate:
 
-A relying party may require additional proof of the proper operation of the issuance log by third parties, like CT logs provide additional transparency. Instead of presenting certificates to transparency logs, CAs present their log state to *cosigners*. Cosigners validate the log is append-only and optionally provide additional services, such as mirroring its contents. They cosign the CA's checkpoints and subtrees.
+5. The CA now has enough information to construct a certificate and give it to the authenticating party. A certificate contains:
 
-A signature from either the CA or a cosigner is known as a *cosignature*. A combination of cosignatures from sufficient trusted parties proves both that the CA has issued the certificate, and that the issuance satisfies any additional relying party transparency requirements.
+   * The TBSCertificate being certified
+   * An inclusion proof from the TBSCertificate to some subtree
+   * Cosignatures from the CA and cosigners
+
+6. As in Certificate Transparency, monitors observe the issuance log to ensure the CA is operated correctly.
+
+A certificate with cosignatures is known as a *full certificate*. Analogous to X.509 trust anchors and trusted CT logs, relying parties are configured with trusted cosigners ({{trusted-cosigners}}) that allow them to accept Merkle Tree certificates. The inclusion proof proves the TBSCertificate is part of some subtree, and cosignatures frm trusted cosigners prove the subtree was certified by the CA and available to monitors. Where CT logs entire certificates, the issuance log's entries are smaller TBSCertificateLogEntry ({{log-entries}}) structures, which do not scale with public key or signature size.
 
-A *full certificate* contains:
+This same issuance process also produces a *signatureless certificate*. This is an optional, optimized certificate that avoids cosignatures. Signatureless certificates are available after a short period of time and usable with up-to-date relying parties.
 
-* The TBSCertificate being certified
-* An inclusion proof from the TBSCertificate to some subtree
-* Sufficient cosignatures to meet the relying party's requirements
+~~~aasvg
++-- Certificate Authority -------+
+|                                |  +-- Update Channel --+
+|    /\                          |  |                    |
+|   /  \  1. Allocate landmarks -+--+----------------+   |
+|  +----+                  |     |  |                |   |
++--------------------------+-----+  +----------------+---+
+                           |                         |
+    2. Make signatureless  |          3. Distribute  |
+       cert from landmark  |              landmarks  |
+                           V                         |
++-- Authenticating Party --------+                   |
+|                                |                   |
+| signatureless cert             |                   V
```

Comment:
Do we want to mention here that a AP will likely have two signature less certs?

Reply:
Hmm. My feeling is it might distract a bit from the core idea.

Reply:
At least in the picture. Maybe I can work it into prose...

Reply:
Added a sentence.